packages/twenty-front/src/modules/object-record/record-table/record-table-header/components/RecordTableColumnHeadWithDropdown.tsx

```diff
@@ -22,8 +24,21 @@ const StyledDropdown = styled(Dropdown)`
 export const RecordTableColumnHeadWithDropdown = ({
   column,
 }: RecordTableColumnHeadWithDropdownProps) => {
+  const { setEnableXScroll, setEnableYScroll } = useContext(
+    RecordTableScrollContext,
+  );
+  const handleDropdownOpen = useCallback(() => {
```

Comment:
Please use recoil states V2 scoped by record table instead of this pattern which we don't use.

See an example here : `isPersistingViewFieldsComponentState.ts`